lib/rufo/new_formatter.rb

```diff
@@ -632,6 +701,38 @@ def visit_def(node)
       visit body
     end
 
+    def visit_do_block(node)
+      # [:brace_block, args, body]
+      _, args, body = node
+
+      line = @line
+
+      consume_keyword "do"
+
+      consume_block_args args
+
+      if body.first == :bodystmt
+        write_breaking_hardline
+        visit_bodystmt body
+      else
+        write_hardline
+        indent_body body
+        write_hardline
+        consume_keyword "end"
+      end
+    end
+
+    def consume_block_args(args)
+      if args
+        consume_one_dynamic_space_or_newline @spaces_around_block_brace
+        # + 1 because of |...|
+        #                ^
+        indent(@column + 1) do
```

Comment:
Does this method have an effect? Based on the fact that `consume_one_dynamic_space_or_newline` isn't defined in our formatter, my guess is that it doesn't get called. Could we remove it?